### ISSUE TITLE
[FR] When printing with no bed temperature, disable warming up for homing

### ISSUE THREAD

**Issue by tobar-io** [CR6Community/Marlin#298]
### Describe the feature you want

When printing TPU, I want the bed to stay cool so I let the slicer write 0 into the GCODE. The CR6SE CF 6.1 homing procedure heats the bed up. After that it lets the bed cool forever. 

My proposal is to implement a check like this:

if the target temperature is 0 -> don't heat up for homing

**Comment by Thinkersbluff** [comment]
Have you tried setting the bed temperature  pre-heat setting to zero?  
![Uploading E92BFC37-1B8A-411F-870E-FC9EA351E8DE.jpeg…]()

**Comment by tobar-io** [comment]
No, but this would be a work around for this particular case of course.

**Comment by ritchiedc** [comment]
Your solution would require a work around for my use case where I don't set temperature until after homing. It's likely it would affect a larger number of people since this method is used by Sebazzz's recommended start gcode. There's always trade-offs.
